Misc/NEWS.d/next/Library/2023-07-15-12-52-50.gh-issue-105726.NGthO8.rst

```diff
@@ -0,0 +1,2 @@
+Bug fixed: Inheritance from AbstractContextManager and AbstractAsyncContextManager
```

Comment:
```suggestion
Added ``__slots__`` to :class:`contextlib.AbstractContextManager` and :class:`context.AbstractAsyncContextManager`
so that child classes can use ``__slots__``.
```